config/routes.rb

```diff
@@ -0,0 +1,15 @@
+Rails.application.routes.draw do
+  namespace :api do
+    namespace :v1 do
+      get 'greetings', to: 'random_greetings#index'
+    end
+  end
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Reveal health status on /up that returns 200 if the app boots with no exceptions, otherwise 500.
+  # Can be used by load balancers and uptime monitors to verify that the app is live.
+  get "up" => "rails/health#show", as: :rails_health_check
+
+  # Defines the root path route ("/")
+  # root "posts#index"
+end
```

Comment:
- [ ] Kindly make sure that the app works as intended. Currently, the front end portion of the app doesn't contact the backend properly. You can see this from the [following line of code in the frontend](https://github.com/jecode93/hello-react-front-end/blob/4d80d4aa2a13b20a0f7b716deadeb3c7ee7c2563/src/redux/greeting/greetingSlice.js#L2-L4). This is not the right URL to contact since these are two separate apps. To fix this simply use the full URL of the backend's API.

- [ ] Kindly fix the following error that appears when the other issue is fixed. You can read more about cors and how to fix it [here](https://medium.com/@chris.lty07/configuring-for-cors-and-cookies-with-react-and-ruby-on-rails-1c30219cc2db).

![image](https://github.com/jecode93/hello-rails-back-end/assets/62290545/d5ae803b-c531-45bd-bffe-61a60b74eaa2)

Reply:
@ShadyShawkat the app works properly on my side, the problem is from your browser bro, the error says Access-control-Allow-Origin...

Can you try it on another browser?